### PR TITLE
multi distros

### PR DESCRIPTION
**Pull request by DenisMishin**
note that Azure doesn’t support cloud-init scripts yet for RHEL/CentOS,
only Ubuntu.

### CODE REVIEW

File: assets/terraform/aws/bootstrap/debian.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+./ubuntu.sh
```
Comment (UnderGreen) [inline assets/terraform/aws/bootstrap/debian.sh]: why not to make symbolic link here?

Comment (DenisMishin) [inline assets/terraform/aws/bootstrap/debian.sh]: this is a symbolic link 
```
$ cd assets/terraform/aws/bootstrap/
$ ls -l
-rw-r--r--  1 denismishin  staff  727 15 May 10:44 centos.sh
lrwxr-xr-x  1 denismishin  staff   11 15 May 10:44 debian.sh -> ./ubuntu.sh
lrwxr-xr-x  1 denismishin  staff   11 15 May 10:44 redhat.sh -> ./centos.sh
-rw-r--r--  1 denismishin  staff  613 15 May 10:44 ubuntu.sh
```

Comment (UnderGreen) [inline assets/terraform/aws/bootstrap/debian.sh]: ahh, it is a github representation of symbolic links :smiley:

File: lib/system/file.go [hunk 2]
```diff
@@ -73,6 +73,24 @@ func CopyAll(src, dst string) (fileCount uint, err error) {
 	return fileCount, err
 }
 
+/*
```
Comment (sofuture) [inline lib/system/file.go]: nuke

**Pull request merged**